src/Tools/dotnet-monitor/CollectionRules/Options/Actions/CollectStacksOptionsDefaults.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.Diagnostics.Tools.Monitor.CollectionRules.Options.Actions
+{
+    internal static class CollectStacksOptionsDefaults
+    {
+        public static readonly CallStackFormat DefaultFormat = CallStackFormat.Json;
```

Comment:
```suggestion
        public static readonly CallStackFormat Format = CallStackFormat.Json;
```

We already know it's the default based on the class name.